src/qpid/broker/SelectorToken.h

```diff
@@ -69,7 +69,8 @@ struct Token {
     std::string val;
     std::string::const_iterator tokenStart;
 
-    Token()
+    Token() :
+        type()
```

Comment:
The base type of an enum is a primitive integral type so it is correct to initialise it here. However "type()" will assign zero to the variable irrespective of whether that is a valid enum element - perhaps better to explicitly assign an enum element eg: type(T_EOS) (if that was the intention) or type(T_NULL).